### DISCUSSION TITLE
Security Onion 2 Booting issue

### DISCUSSION THREAD

**Discussion by Jassecteam** [Security-Onion-Solutions/securityonion#1848] [Unsupported Versions]
Hai,
When boot the security onion 2 ISO using a bootable USB Flash drive created with Balena Etcher.I successfully run the media, then I see a text like this  "starting installer ,one moment...." , after this there is some text that flashes on the screen too quickly and see a blinking cursor. Wait for more than 15 minutes but nothing seen in the screen.
What wrong with this? Is there any bug ?
Please help with a solution.

**Comment by weslambert** [comment]
What kind of hardware/boot mode are you using?
